bin/cron.sh

```diff
@@ -10,12 +9,30 @@ case "$(hostname)" in
         export BUILDKITE_PATH="/clima/slurm-buildkite"
         export BUILDKITE_QUEUE='clima'
         ;;
+    derecho[0-7]|"cron")
```

Comment:
`cron` could be a general name used by other computers as well, which might lead to name collision.

Reply:
I added a check for the BUILDKITE_PATH, but I am not sure how to get identifying information on the derecho cron node.
```
if [[ -d BUILDKITE_PATH ]]; then
    echo "Could not find buildkite dir $BUILDKITE_PATH. Exiting..."
    exit 1
fi
```

Reply:
I don't know either. We can keep it as it, maybe just add a note to draw attention.